### DISCUSSION TITLE
Add link to spacescan for TAIL search

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#2377] [Ideas]
[SpaceScan](https://www.spacescan.io) now supports searching for TAILs, similar to [TAIL Database](https://www.taildatabase.com).
Can we get a second link added to the `MANAGE TOKEN LIST` section of the `Tokens` panel of the GUI? Currently, there is only `Search on Tail Database`.

For example the equivalent links for the Chia Holiday 2021 token are: 
https://www.taildatabase.com/tail/509deafe3cd8bbfbb9ccce1d930e3d7b57b40c964fa33379b18d628175eb7a8f
https://www.spacescan.io/cat2/509deafe3cd8bbfbb9ccce1d930e3d7b57b40c964fa33379b18d628175eb7a8f

**Comment by BrandtH22** [comment]
@danieljperry , note I transferred this enhancement request to the GUI repo for better visibility 👍
